packages/module-utils/src/generate-dts-sourcemaps.ts

```diff
@@ -280,3 +322,20 @@ export function generateDTSSourceMap(dtsContent: string, meta: StylableMeta) {
         4
     );
 }
+
+function calculateEstimatedPosition(
```

Comment:
maybe open an issue (or comment on the value parser issue) about this as a tech debt